adminSiteClient/GdocsPreviewPage.tsx

```diff
@@ -233,9 +234,46 @@ export const GdocsPreviewPage = ({ match, history }: GdocsMatchProps) => {
                     <GdocsDiff originalGdoc={originalGdoc} gdoc={gdoc} />
                 </Drawer>
 
-                <DebugProvider>
-                    <OwidArticle {...gdoc} />
-                </DebugProvider>
+                {/* 
+                    This uses the full SSR rendering pipeline. It is more accurate but comes
+                    with an additional requests to the Google API and has a less polished
+                    authoring experience at the moment (content flashes and scrolling position
+                    resets on every change)
+                */}
+                <iframe
+                    src={`/gdocs/${gdoc.id}/preview#owid-article-root`}
+                    style={{ width: "100%", height: "inherit", border: "none" }}
+                    key={gdoc.revisionId}
```

Comment:
Just checking: this is only needed so that the iframe knows when to refresh?

Reply:
yes, exactly